src/services/folder.rs

```diff
@@ -25,54 +25,40 @@ use crate::utils;
 pub struct FolderService;
 
 impl FolderService {
-    pub async fn get(
-        ctx: Arc<Context>,
-        id: Uuid,
-        _reference: String,
-        path: Option<String>,
-        recursive: Option<&String>,
-    ) -> Result<Tree> {
+    pub async fn get(ctx: Arc<Context>, id: Uuid, path: Option<String>) -> Result<Vec<File>, ApiError> {
```

Comment:
`path: Option<String>` 

After splitting the tree, path is a required field in all Files interfaces.